src/gsi/m_berror_stats_reg.f90

```diff
@@ -870,16 +875,29 @@ subroutine berror_read_wgt_reg(msig,mlat,corz,corp,hwll,hwllp,vz,rlsig,varq,qopt
            hwllp(i,n)=hwllp(i,nrf2_ps)
         end do
      else if (n==nrf2_howv) then
-         call read_howv_stats(mlat,1,2,cov_dum)
+         call read_howv_stats(mlat,1,2,cov_dum,mype)     ! static B for howv
          do i=1,mlat
             corp(i,n)=cov_dum(i,1,1)     !#ww3
             hwllp(i,n) = cov_dum(i,1,2) 
          end do
          hwllp(0,n) = hwllp(1,n)
          hwllp(mlat+1,n) = hwllp(mlat,n)
-
-         if (mype==0) print*, 'corp(i,n) = ', corp(:,n)
-         if (mype==0) print*, ' hwllp(i,n) = ',  hwllp(:,n)
+         if (mype==0) then
+            print*, myname_, ' static BE corp( :,n) (for ', trim(adjustl(cvars2d(n))), ')= ', corp(:,n)
+            print*, myname_, ' static BE hwllp(:,n) (for ', trim(adjustl(cvars2d(n))), ')= ', hwllp(:,n)
+         end if
```

Comment:
suggest to move line 885-888 after line 990 and remove line 897

Reply:
The print-out in lines 885-888 are used to print out the original corp and hwllp (2DRTMA also needs to check them, so in original code there are also two lines to print them.). 
The print-out-lines in if block 890-900 are used to check the corp if the original value is tuned (2DRTMA does not need to run this).

Reply:
@guoqing-noaa 
I am going to remove subroutine get_factor_tuneBE_howv, so there will be no if-block 890-900 to call get_factor_tuneBE_howv. Then no need to move line 885-888 after line 890.
Thanks,